$NetBSD: patch-rebar.config,v 1.2 2017/03/30 14:32:47 fhajny Exp $

Deps are handled by pkgsrc.
Look up fast_xml includes properly.

--- rebar.config.orig	2017-02-23 17:12:15.000000000 +0000
+++ rebar.config
@@ -22,10 +22,7 @@
 {erl_opts, [debug_info, {src_dirs, ["src"]},
 	    {i, "include"},
 	    {i, "../fast_xml/include"},
-	    {i, "deps/fast_xml/include"}]}.
-
-{deps, [{fast_xml, ".*", {git, "https://github.com/processone/fast_xml", {tag, "1.1.22"}}},
-	{stringprep, ".*", {git, "https://github.com/processone/stringprep", {tag, "1.0.8"}}}]}.
+	    {i, "@LOCALBASE@/lib/erlang/lib/fast_xml/include"}]}.
 
 {cover_enabled, true}.
 {cover_export_enabled, true}.
